dist/morphext.js

```diff
@@ -22,6 +22,8 @@
             animation: "bounceIn",
             separator: ",",
             speed: 2000,
+            activeClass: 'active',
+            loop: 10, // loop n times around TBC
```

Comment:
I'm not sure if I'm keen on including an out-of-the-box support for a loop limit. Seems like something that can be achieved by the `complete` callback.